src/Tooltip/Tooltip.js

```diff
@@ -169,7 +187,8 @@ Tooltip.propTypes = {
   onMouseLeave: propTypes.func,
   onOpen: propTypes.func,
   style: propTypes.shape({}),
-  text: propTypes.string.isRequired
+  text: propTypes.string.isRequired,
+  direction: propTypes.oneOf(['up', 'down', 'left', 'right'])
```

Comment:
I think its better to name the prop `direction` and make it accept values `top, bottom, left, right`